core/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js

```diff
@@ -401,7 +402,11 @@ oppia.controller('CreatorDashboard', [
             // the scope (the property cannot sit directly on the scope)
             // Reference https://stackoverflow.com/q/12618342
             $scope.suggestionData = {newSuggestionHtml: newContent.html};
-            $scope.suggestionEditorIsShown = false;
+            if (EditButtonShow === 0){
```

Comment:
Don't check === 0, you can just say `if (!variable)`.

Reply:
ok sir